### PR TITLE
dasharo-compatibility/306-usb-hid-and-msc-support.md: update description tests USB001.003 and USB002.003

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md [hunk 1]
```diff
@@ -210,16 +225,44 @@ by the `OPERATING_SYSTEM` and all basic keys work according to their labels.
...
+    ```
+
+1. Note results.
 
 **Expected result**
 
-1. All standard keyboard keys generate correct characters
-   or actions when pressed.
-1. Key combinations are detected correctly.
+Informations about connected keyboard should be displayed.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:238]:
```suggestion
1. Information about connected keyboard should be displayed.
```

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md [hunk 2]
```diff
@@ -103,15 +103,30 @@ by the `OPERATING_SYSTEM` and all basic keys work according to their labels.
...
+    OK         USB             Generic USB Hub
+    OK         HIDClass        USB Input Device
+    OK         Bluetooth       Intel(R) Wireless Bluetooth(R)
+    OK         USB             USB Root Hub (USB 3.0)
+    OK         Net             TP-LINK Gigabit Ethernet USB Adapter
+    OK         USB             Generic USB Hub
+    OK         USB             USB Mass Storage Device
+    ```
+
+Connect additional USB device, run command once again and note reults.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:128]:
```suggestion
Connect additional USB device, run command once again and note the results.
```

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md [hunk 3]
```diff
@@ -103,15 +103,30 @@ by the `OPERATING_SYSTEM` and all basic keys work according to their labels.
...
     ```
 
-1. Connect external USB devices to DUT USB A port and note the result.
+1. Note results.
 
 **Expected result**
 
-1. After each device is connected to the USB port, a new USB device entry in the
-    output of the command should appear.
+Example output:
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:113]: Indentation missing

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md [hunk 4]
```diff
@@ -103,15 +103,30 @@ by the `OPERATING_SYSTEM` and all basic keys work according to their labels.
 1. Open PowerShell and and run the follwing command:
 
     ```bash
-    while (1) {Get-PnpDevice -PresentOnly | Where-Object { $_.InstanceId -match '^USB' }; sleep 5; clear}
+    Get-PnpDevice -PresentOnly | Where-Object { $_.InstanceId -match '^USB' }
     ```
 
-1. Connect external USB devices to DUT USB A port and note the result.
+1. Note results.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:109]:
```suggestion
1. Note the results.
```

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:128]: indentation missing

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:238]: indentation missing

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md:238]: add all improvements

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**